packages/expo-device/ios/EXDevice/DeviceUID.m

```diff
@@ -0,0 +1,148 @@
+//
```

Comment:
Remove this and implementation file

Reply:
This whole class seems unnecessary -- just use `identifierForVendor` which we can reliably call.